spec/ttml1.xml

```diff
@@ -4277,20 +4277,20 @@ as a style property only to those element types indicated in the following table
 </tr>
 <tr>
 <td><emph>Percentages:</emph></td>
-<td>if not <loc href="#content-vocabulary-span"><el>region</el></loc> element,
-then relative to parent element's font size; otherwise, relative to the
-<emph>Computed Cell Size</emph></td>
+<td>see discussion below</td>
 </tr>
 <tr>
 <td><emph>Animatable:</emph></td>
 <td>discrete</td>
 </tr>
 </tbody>
 </table>
+<p>The computed value of the property associated with this attribute consists of a 2-tuple
+the entries of which denote the computed values of the width and height components of the font size respectively.</p>
 <p>If a single <loc href="#style-value-length">&lt;length&gt;</loc> value is specified, then this length applies
-equally to horizontal and vertical scaling of a glyph's EM square; if two
+equally to horizontal and vertical size of a glyph's EM square; if two
 <loc href="#style-value-length">&lt;length&gt;</loc> values are specified, then the first expresses the horizontal
-scaling and the second expresses vertical scaling.</p>
+size and the second expresses vertical size.</p>
```

Comment:
Proposal:

_Note: Presentation processors are expected to scale glyphs so that the height of the em square as defined in the font resource matches the computed font size height, using an appropriate algorithm for the font. If the computed font size width differs from the computed font size height presentation processors additionally need to scale the em square anamorphically by the ratio of the width to the height._

Reply:
Action on me to file this against TTML2 for discussion there.

Reply:
Filed as #547